samples/sample_decode/src/pipeline_decode.cpp

```diff
@@ -1884,6 +1904,36 @@ mfxStatus CDecodingPipeline::RunDecoding()
                 bErrIncompatibleVideoParams = true;
                 // need to go to the buffering loop prior to reset procedure
                 pBitstream = NULL;
+                sts = MFX_ERR_NONE;
+                continue;
+            } else if (MFX_ERR_REALLOC_SURFACE == sts) {
+                mfxVideoParam param;
+                m_pmfxDEC->GetVideoParam(&param);
+
+                m_pCurrentFreeSurface->frame.Info.CropW = param.mfx.FrameInfo.CropW;
+                m_pCurrentFreeSurface->frame.Info.CropH = param.mfx.FrameInfo.CropH;
+                m_YUV_Width = MSDK_ALIGN16(std::max(param.mfx.FrameInfo.Width, (mfxU16)m_YUV_Width));
```

Comment:
Use MSDK_MAX instead of std::max here to avoid windows issues.

Reply:
Sure. I will change it.